setup.py

```diff
@@ -21,28 +21,28 @@
     zip_safe=False,
     packages=find_packages(),
     install_requires=[
-        "gym>=0.26",
-        "numpy>=1.20.0",
-        "GitPython",
-        "transforms3d>=0.3.1",
-        "networkx>=2.0",
-        "PyYAML",
-        "addict",
-        "ipython",
-        "future",
-        "trimesh",
-        "h5py",
-        "cryptography",
+        "gym~=0.26",
+        "numpy~=1.23",
+        "GitPython~=3.1",
+        "transforms3d~=0.4",
+        "networkx~=3.1",
+        "PyYAML~=6.0",
+        "addict~=2.4",
+        "ipython~=8.18",
+        "future~=0.18",
+        "trimesh~=4.0",
+        "h5py~=3.10",
+        "cryptography~=41.0",
         "bddl~=3.1.0",
-        "opencv-python",
-        "nest_asyncio",
-        "imageio",
-        "imageio-ffmpeg",
-        "termcolor",
-        "progressbar",
-        "pymeshlab",
-        "click",
-        "aenum",
+        "opencv-python~=4.8",
+        "nest_asyncio~=1.5",
+        "imageio~=2.22",
+        "imageio-ffmpeg~=0.4",
+        "termcolor~=2.4",
+        "progressbar~=2.5",
+        "pymeshlab~=2023.12",
```

Comment:
I dont think this version works on Windows? Or have you fixed that? Otherwise maybe we can try the latest 2022

Reply:
Just tested on windows and OG seems working fine